lib/krikri/harvesters/fedora_harvester.rb

```diff
@@ -0,0 +1,158 @@
+require 'uri'
+require 'async_uri_getter'
+
+module Krikri::Harvesters
+  ##
+  # A harvester implementation for Fedora 3
```

Comment:
We weren't totally clear about how harvesters should be split between Heidrun and KriKri, so this was something we'd intended to check.  I had the sense that generic harvesters lived in KriKri, while site-specific ones lived in Heidrun, so perhaps IA and NYPL should move to Heidrun as well?

For the sake of this PR we just kept them all together, but can easily move them around.